build/helper/documentation_helper.py

```diff
@@ -123,7 +123,7 @@ def get_rst_admonition_snippet(node, admonition, d, config, indent=0):
 
 def add_attribute_rep_cap_tip_rst(attr, config):
     '''Add the appropriate (r/w/rw/none) rst formatted tip for an attribute'''
-    if 'repeated_capability_type' in attr:
```

Comment:
This key is used in [nidigital and nifgen](https://github.com/search?q=repeated_capability_type+repo%3Ani%2Fnimi-python+filename%3Aattributes.py&type=Code&ref=advsearch&l=&l=). I'm surprised changing this didn't update their generated code and docs.

Reply:
It did. I manually removed the generated files of the other drivers from the commit.

Even though I've already updated the metadata for the other drivers, I didn't include the output from the metadata exports of the other drivers (e.g. attributes.py) as inputs to the code generation in this repo, and so the code generated files would have been incorrect.

Reply:
Ok, Please have a work item in your backlog to update the metadata and generated files for those drivers.

Reply:
Does this mean that should someone clone the repo and run codegen would see generated files that are different from what was committed? That's not a state we want to leave the repo in. (unresolved until clarification)

Reply:
Yes, @marcoskirsch, you are correct. Your scenario would happen as you described.

I left out the digital, dmm, fgen, modinst, scope, switch and tclk files from this PR because I assumed we wouldn't want to include diffs in this PR belonging to drivers, unrelated to the issue being addressed by this PR, and also because the internal nidigital build has been broken for a while (not sure it if still is) and so it wouldn't have been possible for me to use the metadata changes that I made for those drivers in the codegen build for this PR. Until that happens, I can't generate correct versions of at least the digital and scope files.

So now I see 3 possibilities. Let me know which one you prefer:

1. Block this PR until the nidigital build is fixed, and then include all the other updated driver files as part of this PR
2. Submit this PR as is, wait for the nidigital build to be fixed, and then immediately implement the user story that @sbethur ask that I create, which captured the task of updating those other files. This will result in the repo having latent, pending changes as you pointed out, at least for a couple of days, until the next PR goes through.
3. Include in this PR the version of the files that are currently being generated, which are incorrect for digital and scope. Similarly to item 2, this would be fixed in a few days. It's only the generated documentation that is incorrect, so maybe that's not so terrible.

I had implicitly chosen option 2, based on the fact that changes have gone into this repo in past that consisted of doing direct edits on the generated files, which also triggers the same scenario that you described. And in this case, we'd be in that state for a shorter amount of time. But if you guys prefer one of the other options, or know of a different option, let me know, and I'll be happy to comply.

Reply:
> based on the fact that changes have gone into this repo in past that consisted of doing direct edits on the generated files

Really? That is pretty bad. Or are you talking about the metadata files generated internally at NI and uploaded to GitHub?

The problem with committing like this is that the files will show up as noise to every other developer.
@sbethur discussed and we think option 3 is best. Documentation will be temporarily wrong but is possibly the lesser of 3 evils.

Reply:
Sorry, yes, I was referring to editing the metadata files (functions.py, etc...), rather than changing their source hapigen files. So I guess cloning the repo and rebuilding wouldn't have resulted in changes. Sorry for conflating the two.